bin/local-env/start.sh

```diff
@@ -9,12 +9,6 @@ set -e
 # Change to the expected directory
 cd "$(dirname "$0")/../.."
 
-# Check whether Node and NVM are installed
-. "$(dirname "$0")/install-node-nvm.sh"
-
-# Check whether Composer installed
-. "$(dirname "$0")/install-composer.sh"
-
```

Comment:
No longer needed?

Reply:
No, the CI runners already have these tools pre-installed, and we configure NodeJS and Composer in the CI. 

For local development, I doubt if this is being used given we already have wp-env setup.